app/validation.py

```diff
@@ -115,6 +125,22 @@ def validates_against_schema(validator_name, submitted_json):
         return True
 
 
+def get_validation_errors(validator_name, json_data, enforce_required=True):
+    error_map = {}
+    try:
+        validator = get_validator(validator_name, enforce_required)
+        errors = sorted(validator.iter_errors(json_data), key=lambda e: e.path)
+        for index, error in enumerate(errors):
+            if error.path:
+                key = error.path.pop()
+            else:
+                key = "err_{}".format(index)
```

Comment:
What's the reason behind the `err_%d` format? It seems like it'd be harder to use than a JSON list of errors.